src/components/Header/Header.tsx

```diff
@@ -1,10 +1,42 @@
-import { Button, Flex } from "@adobe/react-spectrum";
+import { ActionButton, Button, Flex, Item, Menu, MenuTrigger } from "@adobe/react-spectrum";
```

Comment:
the new components as to implement the pretty menu, will check on the documentation for MenuTrigger and Item